### ISSUE TITLE
When playing 'cue' sheet, the album art of the corresponding 'flac' is not being displayed.

### ISSUE THREAD

**Issue by alexvong1995** [mpv-player/mpv#5155]
### mpv version and platform

```
alexvong1995@debian:~$ mpv -V
mpv 0.27.0 (C) 2000-2017 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
ffmpeg library versions:
   libavutil       55.78.100
   libavcodec      57.107.100
   libavformat     57.83.100
   libswscale      4.8.100
   libavfilter     6.107.100
   libswresample   2.9.100
ffmpeg version: 3.4
```

### Reproduction steps

Play any `cue` sheet with the corresponding `flac` having an album cover.

### Expected behavior

The album cover should be displayed.

### Actual behavior

The album cover is not being displayed.

### Log file

https://notabug.org/alexvong1995/mpv-log/src/master/mpv-cue-log

### Sample files

Will be provided upon request.

Feel free to ask for more info if needed. Thank you!

**Comment by thebombzen** [comment]
I can't reproduce this, i.e. it works as expected for me.

**Comment by kevmitch** [comment]
Please upload a sample to https://0x0.st/.

**Comment by alexvong1995** [comment]
Kevin Mitchell <notifications@github.com> writes:

> Please upload a sample to https://0x0.st/.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

Sorry, I am busy right now. I will upload a sample next weekend.

**Comment by alexvong1995** [comment]
Alex Vong <alexvong1995@gmail.com> writes:

[...]
>
> Sorry, I am busy right now. I will upload a sample next weekend.

Sorry for the long delay. I have uploaded the sample to
https://notabug.org/alexvong1995/mpv-log/src/master/sample/.

You can clone the repo if you like.

To reproduce the issue, go to the directory `sample/`.
If you run `mpv noise.cue`, no album art should be displayed.
If you run `mpv noise.flac`, an album art should be displayed.

Thank you.

**Comment by kevmitch** [comment]
fixed in 3d32cb164957854f84e41d9aeb62e0ea185d0ca9
